events/static/events/create_event.css

```diff
@@ -230,3 +229,12 @@ a {
 .fa-file:hover {
     color: #aae0f0;
 }
+
+::-webkit-scrollbar {
+  width: 10px;
+}
+
+::-webkit-scrollbar-thumb {
+  background: #aae0f0;
+  border-radius: 10px;
+}
```

Comment:
Please fix the indentation in this file 😄

Reply:
done. thanks 😄